tests/unit/test_charm.py

```diff
@@ -1284,3 +1284,25 @@ def test_on_config_changed_with_invalid_log_level(harness: Harness) -> None:
 
     assert isinstance(harness.model.unit.status, BlockedStatus)
     assert "Invalid configuration value for log_level" in harness.charm.unit.status.message
+
+
+def test_on_pebble_ready_make_dir_called(
+    harness: Harness, mocked_isdir: MagicMock, mocked_make_dir: MagicMock
+) -> None:
+    container = harness.model.unit.get_container(CONTAINER_NAME)
+    harness.charm.on.kratos_pebble_ready.emit(container)
+
+    mocked_isdir.assert_called_once_with("/var/log")
+    mocked_make_dir.assert_called_once_with(path="/var/log", make_parents=True)
```

Comment:
I think this is cleaner:
```suggestion
def test_on_pebble_ready_make_dir_called(harness: Harness) -> None:
    container = harness.model.unit.get_container(CONTAINER_NAME)
    harness.charm.on.kratos_pebble_ready.emit(container)

    assert container.isdir("/var/log")
```

Reply:
done